### ISSUE TITLE
Can't use free account

### ISSUE THREAD

**Issue by Watson1978** [HipByte/motion-provisioning#18]
* Result
```
$ rake
License file not present. Please activate RubyMotion with `motion activate' and try again.
Couldn't verify that spaceship is up to date
      Info Output directory for MotionProvisioning set to '/Users/watson/prj/RubyMotionSamples/ios/SpaceRocks/provisioning'.
      Info Logging into the Developer Portal with email 'watson1978@gmail.com'.
rake aborted!
User 'watson1978@gmail.com' does not have access to any teams with an active membership
/Users/watson/prj/motion-provisioning/lib/motion-provisioning.rb:98:in `client'
/Users/watson/prj/motion-provisioning/lib/motion-provisioning/certificate.rb:6:in `client'
/Users/watson/prj/motion-provisioning/lib/motion-provisioning/certificate.rb:39:in `certificate_name'
/Users/watson/prj/motion-provisioning/lib/motion-provisioning.rb:207:in `certificate'
/Users/watson/prj/RubyMotionSamples/ios/SpaceRocks/Rakefile:15:in `block in <top (required)>'
/Users/watson/prj/RubyMotion/lib/motion/project/config.rb:121:in `block in setup'
/Users/watson/prj/RubyMotion/lib/motion/project/config.rb:121:in `each'
/Users/watson/prj/RubyMotion/lib/motion/project/config.rb:121:in `setup'
/Users/watson/prj/RubyMotion/lib/motion/project/app.rb:66:in `config'
/Users/watson/prj/RubyMotion/lib/motion/project/template/ios.rb:156:in `block in <top (required)>'
Tasks: TOP => default => simulator
(See full trace by running task with --trace)
```

* Configuration
```
Motion::Project::App.setup do |app|
...

  app.codesign_certificate = MotionProvisioning.certificate(
    type: :development,
    platform: :ios,
    free: true
  )
  app.provisioning_profile = MotionProvisioning.profile(
    bundle_identifier: app.identifier,
    app_name: app.name,
    platform: :ios,
    type: :development,
    free: true
  )
end
```

**Comment by andrewhavens** [comment]
This was fixed in version 1.0.0.

**Comment by Watson1978** [comment]
Thanks
